staging/src/k8s.io/kubectl/pkg/cmd/top/top_node.go

```diff
@@ -188,13 +188,13 @@ func (o TopNodeOptions) RunTopNode() error {
 		nodes = append(nodes, nodeList.Items...)
 	}
 
-	allocatable := make(map[string]v1.ResourceList)
+	capacity := make(map[string]v1.ResourceList)
 
 	for _, n := range nodes {
-		allocatable[n.Name] = n.Status.Allocatable
+		capacity[n.Name] = n.Status.Capacity
```

Comment:
The reason we pick `Allocatable` over `Capacity` is because it represents the value available for scheduling, ie. for running pods. That's by design, not a bug.